src/capi/topology.cpp

```diff
@@ -107,7 +107,6 @@ extern "C" chfl_status chfl_topology_impropers_count(const CHFL_TOPOLOGY* const
     )
 }
 
-#pragma intel optimization_level 2  /* Using -O3 with icc lead to partial copy of the bonds */
```

Comment:
So they are not needed after all!

Reply:
Nope! This was pretty old though, I don't remember why I had to add them